Dockerfile.final

```diff
@@ -0,0 +1,3 @@
+ARG TAG=main
+ARG APP=ui
+FROM ghcr.io/city-of-helsinki/tilavarauspalvelu-ui-$APP:$TAG AS $APP
```

Comment:
Should this be `tilavarauspalvelu-$APP` instead?

Reply:
No because of the way the name is created
`echo ${{ env.REGISTRY }}/${{ env.REPO_NAME }}-${{ env.BUILD_APP }} | tr '[:upper:]' '[:lower:]'`

Otherwise have to do some regex / string magic to strip the `-ui` part of the repo name which I think is more fragile than repeating. 

The package names always include `tilavarauspalvelu-ui` (as per repo name) and the app name and we end up with `tilavarauspalvelu-ui-ui` and `tilavarauspalvelu-ui-admin-ui`. I'd rather rename the apps than not use the repo name as part of the package name.